tempodb/encoding/vparquet/block_search_test.go

```diff
@@ -391,3 +392,50 @@ func makeTraces() ([]*Trace, map[string]string) {
 
 	return traces, attrVals
 }
+
+func BenchmarkBackendBlockSearch(b *testing.B) {
+	testCases := []struct {
+		name string
+		tags map[string]string
+	}{
+		{"noMatch", map[string]string{"foo": "bar"}},
+		{"partialMatch", map[string]string{"foo": "bar", "component": "gRPC"}},
+	}
+
+	ctx := context.TODO()
+	tenantID := "1"
+	blockID := uuid.MustParse("3685ee3d-cbbf-4f36-bf28-93447a19dea6")
+
+	r, _, _, err := local.New(&local.Config{
+		Path: path.Join("/Users/marty/src/tmp/"),
```

Comment:
Do we need a tmp dir?

Reply:
This benchmark is designed to run against a block on your local filesystem and that lets us test more realistically than with synthetic blocks created in a temp directory.    I've found these tests useful and was on the fence whether they should be committed, but overall seems worth it. Definitely open to ideas on how to make it more generalized for everyone.

Reply:
At least it's not `C:\Documents and Settings\marty\...` ;)